src/pages/storage/forms/StorageVolumeFormSnapshots.tsx

```diff
@@ -1,79 +1,108 @@
 import React, { FC } from "react";
-import { Input } from "@canonical/react-components";
+import { Icon, Input, Notification } from "@canonical/react-components";
 import { FormikProps } from "formik/dist/types";
 import ConfigurationTable from "components/ConfigurationTable";
 import { getConfigurationRow } from "components/ConfigurationRow";
 import { StorageVolumeFormValues } from "pages/storage/forms/StorageVolumeForm";
 import SnapshotScheduleInput from "components/SnapshotScheduleInput";
 import { useDocs } from "context/useDocs";
+import { useProject } from "context/project";
+import { isSnapshotsDisabled } from "util/snapshots";
+import { useAuth } from "context/auth";
+import { Link } from "react-router-dom";
 
 interface Props {
   formik: FormikProps<StorageVolumeFormValues>;
 }
 
 const StorageVolumeFormSnapshots: FC<Props> = ({ formik }) => {
   const docBaseLink = useDocs();
+  const { project } = useProject();
+  const snapshotDisabled = isSnapshotsDisabled(project);
+  const { isRestricted } = useAuth();
+
+  const snapshotDisabledWarningLink = !isRestricted ? (
```

Comment:
This warning, toghether with the useAuth can go into a dedicated component, so we don't have to repeat it twice. Also the condition should be turned around, so we don't neglect `isRestricted` for the check

Reply:
thank you for the suggestions, I've made those changes :+1: